config/boot.rb

```diff
@@ -1,4 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+# require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
```

Comment:
I noticed we commented out the bootsnap gem and the setup here — is that required for the S3 stuff?

Reply:
I am 👍 on omitting bootsnap for the time being. bootsnap is a caching mechanism to speed up start times. As a cache, it introduces all of the weirdnesses of caching when it is not clear that Terrastories has a problem bootsnap caching solves.

Reply:
Works for me! One fewer dependencies